.vscode/settings.json

```diff
@@ -23,7 +23,7 @@
     "editor.defaultFormatter": "dbaeumer.vscode-eslint"
   },
   "[typescriptreact]": {
-    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
```

Comment:
I noticed this while editing. I was wondering if we should ignore .vscode

Reply:
The value was correct. We're using eslint to format the code (prettier is used as part of eslint).